CHANGES/479.feature

```diff
@@ -0,0 +1,5 @@
+Updated the image building feature:
+* Replaced `artifacts` by `build_context` (i.e., a file plugin repository version HREF)
+as the parameter to provide the build context for a Containerfile.
+* Added the `containerfile_name` as the parameter to provide the relative-path of a File Content
+from the provided `build_context`.
```

Comment:
@pedro-psb, is this bullet-point list in the changelog going to be properly rendered by mkdocs?

Reply:
If the bullet paragraph has a 4 space indentation it will render correctly.

```
Updated the image building feature:
    * Replaced `artifacts` by `build_context` (i.e., a file plugin repository version HREF)
    as the parameter to provide the build context for a Containerfile.
    * Added the `containerfile_name` as the parameter to provide the relative-path of a File Content
    from the provided `build_context`.
```

Otherwise it'll become a flat list.